js/memoryMonitor.js

```diff
@@ -49,14 +51,13 @@ function optimizeMemory() {
 
     const memoryConsumed = (preloadMemory.memoryInfo && preloadMemory.memoryInfo.workingSetSize / 1024) || 0;
     const cpuUsagePercentage = preloadMemory.cpuUsage.percentCPUUsage;
-    const activeNetworkRequest = preloadMemory.activeRequests === 0;
 
     if (memoryConsumed > maxMemory
         && cpuUsagePercentage <= cpuUsageThreshold
         && !isInMeeting
         && getIsOnline()
         && canReload
-        && activeNetworkRequest
+        && networkRequestCount <= 0
```

Comment:
Is this check inverted?

Reply:
@yschimke No, This block of code should be executed only if there is no active network request